config/software/datadog-agent.rb

```diff
@@ -1,60 +1,58 @@
 name 'datadog-agent'
 
-local_agent_repo = ENV['LOCAL_AGENT_REPO']
+local_agent_repo = ENV["LOCAL_AGENT_REPO"]
```

Comment:
why the stylistic change? Single quotes for non-interpolated strings actually have a performance benefit.